GitUpKit/Views/GIQuickViewController.h

```diff
@@ -16,7 +16,18 @@
 #import "GIViewController.h"
 
 @class GCHistoryCommit;
+@class GCCommit;
+@protocol GIQuickViewControllerDelegate
+- (void)quickViewWantsToShowSelectedCommitsList:(NSArray <GCHistoryCommit *> *)commitsList selectedCommit:(GCHistoryCommit *)commit;
+- (void)quickViewDidSelectCommit:(GCHistoryCommit *)commit commitsList:(NSArray <GCHistoryCommit *>*)commitsList;
+@end
 
 @interface GIQuickViewController : GIViewController
 @property(nonatomic, strong) GCHistoryCommit* commit;
 @end
+
+@interface GIQuickViewControllerWithCommitsList : GIViewController
+@property(nonatomic, strong) GCHistoryCommit* commit;
+@property(weak, nonatomic) id<GIQuickViewControllerDelegate> delegate;
+@property(nonatomic, strong) NSArray <GCHistoryCommit *> *list;
+@end
```

Comment:
Also move these into their own .h/.m files. We shouldn't have multiple view controllers in the same file.